packages/gcompat/APKBUILD

```diff
@@ -0,0 +1,32 @@
+#Contributor:
+# Maintainer:
```

Comment:
Could you become the maintainer of this package?

Reply:
Yes, sure!

Reply:
Please could you put your name/email at line 2?